src/components/Header.tsx

```diff
@@ -27,9 +27,36 @@ const queryHeader = () => {
         navigation_menu {
           label
           page_reference {
-            title
-            url
-            uid
+            ... on Contentstack_page {
```

Comment:
why do we need this reference changes??. I tried app locally it is breaking because of this. When I removed it is working